spec/message_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'message'
+require 'database_helpers'
+
+describe Message do
+  describe '.all' do
+    it ' returns a list of messages' do
+      connection = PG.connect(dbname: 'chitter_test')
+
+      #Add the test data
+      message = Message.create(message: "This is my first peep", name: "Joe Bloggs")
+      Message.create(message: "This is my second peep", name: "Joe Bloggs")
+      Message.create(message: "This is my third peep", name: "Joe Bloggs")
+      # connection.exec("INSERT INTO chitter (message) VALUES('This is my first peep');")
+      # connection.exec("INSERT INTO chitter (message) VALUES('This is my second peep');")
+
+      messages = Message.all
+
+      expect(messages.length).to eq 3
```

Comment:
as a rule I'd suggest only having one expectation for each of your unit tests.  The reason for this being that if you later make some change that causes this test to break, you won't be able to tell straight away which expectation is causing the failure.